.github/ISSUE_TEMPLATE/release-blocker.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Release blocker
+about: An issue with critical priority that should be considered before the next release
+title: ''
+labels: ["bug", "release-blocker"]
+assignees: 'adiroiban'
+
+---
+
+Report any issue that should block the next release.
+This should follow the same process are reporting a bug, the only difference is that it will get higher priority.
+
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+
+Steps to reproduce the behavior.
```

Comment:
````suggestion
**Describe how to cause this behavior**

What did you do to get it to happen?
Does it happen every time you follow these steps, sometimes, or only one time?

```
If you have a traceback, please paste it here; otherwise, delete this.
```

````